autoload/vital/__vital__/App/WindowSelector.vim

```diff
@@ -44,6 +54,72 @@ function! s:select(winnrs, ...) abort
   endtry
 endfunction
 
+let s:_popup_winids = []
+
+if has('nvim')
+  function! s:_clear_popups() abort
+    for winid in s:_popup_winids
+      call nvim_win_close(winid, 1)
+    endfor
+    let s:_popup_winids = []
+  endfunction
+
+  function! s:_popup(winnrs, chars) abort
+    for idx in range(len(a:winnrs))
+      let winnr = a:winnrs[idx]
+      let char = a:chars[idx]
+      let [width, height] = [(winwidth(winnr) - len(char) -2)/2, (winheight(winnr) - 1)/2]
+
+      let bufnr = nvim_create_buf(0, 1)
+      let text = printf(' %s ', char)
+
+      call nvim_buf_set_lines(bufnr, 0, -1, 1, [text])
+      let opt = {
+            \ 'relative': 'win',
+            \ 'win': win_getid(winnr),
+            \ 'width': len(text),
+            \ 'height': 1,
+            \ 'col': width,
+            \ 'row': height,
+            \ 'anchor': 'NE',
+            \ 'style': 'minimal',
+            \ 'border': map(['╭', '─', '╮', '│', '╯', '─', '╰', '│'], { _, v -> [v, 'NormalFloat'] }),
```

Comment:
Please make this border characters customizable.

Reply:
Please add `'focusable': v:false` to make sure that the float window is not focusable.

Reply:
add "popup_borderchars" option